tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -31,29 +48,81 @@ public void processFeature(SourceFeature sourceFeature, FeatureCollector feature
         .setAttrWithMinzoom("bridge", sourceFeature.getString("bridge"), 12)
         .setAttrWithMinzoom("tunnel", sourceFeature.getString("tunnel"), 12)
         .setAttrWithMinzoom("layer", sourceFeature.getString("layer"), 12)
-        .setAttrWithMinzoom("oneway", sourceFeature.getString("oneway"), 14)
-        .setAttr("ref", sourceFeature.getString("ref"));
```

Comment:
We don't always want export the `ref` for all zooms... instead we do it by highway types (below). This reduces tile size.